public/docs/ts/latest/guide/forms.jade

```diff
@@ -479,12 +486,11 @@ figure.image-display
     Why "ngForm"? 
     A directive's [exportAs](../api/core/DirectiveMetadata-class.html#!#exportAs) property 
     tells Angular how to link the reference variable to the directive.
-    We set `name` to `ngForm` because the `NgControlName` directive's `exportAs` property happens to be "ngForm".
+    We set `name` to `ngForm` because the `ngModel` directive's `exportAs` property happens to be "ngForm".
+
+    The `exportAs` naming of form directives has been a source of confusion, so very soon, Angular will be renaming the `exportAs` for `ngModel` to 'ngModel'.
     
-    This seems unintuitive at first until we realize that *all* control directives in the 
-    Angular form family &mdash; including `NgForm`, `NgModel`, `NgControlName` and `NgControlGroup` &mdash; *exportAs* "ngForm"
-    and we only ever apply *one* of these directives to an element tag.
-    Consistency rules!
+    This mean we will be able to use the more intuitive `#name=ngModel` when accessing the underlying Angular control.
```

Comment:
@filipesilva It looks like this was included in rc3, so we have to update the template to export ngModel instead of ngForm for the input tags. We should also remove the part that says it will be here soon since it's already here :-)